src/aktualizr_lite/main.cc

```diff
@@ -240,6 +240,9 @@ static int daemon_main(LiteClient &client, const bpo::variables_map &variables_m
     }
 
     client.primary->reportNetworkInfo();
+    if (client.config.telemetry.report_network) {
+      client.primary->reportHwInfo();
```

Comment:
Hm, so you are checking for report_network to report hwinfo? Also wonder why the same check is not done for reportNetworkInfo as well.

Reply:
Got from the commit message, but looks confusing from the code. Mind adding a comment like done for the commit message here?

Reply:
good point. I've added a comment.

Reply:
Looks like minor typo in the comment.. this is really just a nit:
`telemetry.report_networ` -> `telemetry.report_network`